posthog/event_usage.py

```diff
@@ -206,6 +206,22 @@ def report_organization_deleted(user: User, organization: Organization):
     )
 
 
+def report_first_ingestion_reminder_email_sent(user: User):
+    posthoganalytics.capture(
+        user.distinct_id,
+        "first ingestion reminder email sent",
+        groups={"organization": str(user.current_organization.id), "instance": SITE_URL,},
+    )
+
+
+def report_second_ingestion_reminder_email_sent(user: User):
```

Comment:
NIT: Thoughts on parametrizing this? Repetitive structure that would eventually get refactored

Reply:
Yeah I thought about that when I wrote it but I think having it as a separate event will make it easier to measure/track on the app? 😁

Reply:
and if we end up having more ingestion reminder emails we should definitely refactor it !